components/raftstore/src/store/peer_storage.rs

```diff
@@ -213,6 +226,30 @@ impl EntryCache {
     }
 }
 
+impl Default for EntryCache {
+    fn default() -> Self {
+        let cache = VecDeque::default();
+        RAFT_ENTRIES_CACHES_GAUGE.add(cache.capacity() as i64);
+        EntryCache { cache }
+    }
+}
+
+impl Drop for EntryCache {
+    fn drop(&mut self) {
+        RAFT_ENTRIES_CACHES_GAUGE.sub(self.get_mem_size());
+    }
+}
+
+fn update_raft_entries_caches_gauge(old_size: i64, new_size: i64) {
+    let size_change = old_size - new_size;
```

Comment:
I think size change = new size - old size

Reply:
I think both right, decide by the follow code. You can ref my comment.

Reply:
Refactored.